.github/workflows/checks.yaml

```diff
@@ -47,7 +46,7 @@ jobs:
       - name: Install Rust
         uses: dtolnay/rust-toolchain@stable
         with:
-          toolchain: stable
+          toolchain: nightly
 
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2
```

Comment:
Not to self, this Rust Cache might be getting "cache busted" when a branch uses the nightly packages and the `main` branch uses stable